src/interface/src/app/map/map.component.ts

```diff
@@ -41,28 +41,38 @@ export class MapComponent implements AfterViewInit {
 
   constructor(private boundaryService: BoundaryService, private popupService: PopupService) {}
 
-  // // TODO(elsieling) Port data from django backend to this component
-  // private initCalMapperLayer() {
-  //   var data = JSON.parse(existing_projects)
+  private async initCalMapperLayer(layers: L.Control.Layers) {
```

Comment:
A comment explaining (in brief) what this function does and what the CalMapper layer is would be helpful, I think.